README.md

```diff
@@ -138,6 +138,14 @@ try {
 }
 ```
 
+### Sign Transaction Toast
+
+`@perawallet/connect` informs users with toast when users try to sign transaction. If dApp wants to disable it, it can be disabled as follows. The default value of `shouldShowSignTxnToast` is `true`.
```

Comment:
Let's move this paragraph under the `Sign Transaction` title after the first paragraph.

---

I'd also change a bit the paragraph


`@perawallet/connect` guides users with a toast message when the `signTransaction` is triggered. It's enabled by default but in some cases, you may not need to the toast message (e.g. you already have signing guidance for users). There's a configuration called `shouldShowSignTxnToast` to disable it, see the example below

```js
const peraWallet = new PeraWalletConnect({shouldShowSignTxnToast: false});
```